index.js

```diff
@@ -38,11 +41,52 @@ app.post('/posts/:id/comments', (req, res) => {
 });
 
 app.get('/', (req, res) => {
+  console.log(req.session);
   db.Post.findAll({ order: [['createdAt', 'DESC']] }).then((blogPosts) => {
```

Comment:
when we run the SELECT query to the database database adds ORDER BY sql statement to the query, thus our result is ordered by createdAt of the records